code/obj/item/toys.dm

```diff
@@ -106,12 +106,18 @@
 	message = trim(copytext(sanitize(html_encode(message)), 1, MAX_MESSAGE_LEN))
 	if (!message || get_dist(src, user) > 1)
 		return
-	logTheThing("say", user, null, "makes [src] say,  \"[message]\"")
+	logTheThing("say", user, null, "makes [src] say, \"[message]\"")
 	user.audible_message("<span class='emote'>[src] says, \"[message]\"</span>")
 	var/mob/living/carbon/human/H = user
 	if (H.sims)
 		H.sims.affectMotive("fun", 1)
 
+/obj/item/toy/plush/attack(mob/M as mob, mob/user as mob)
+	if (user.a_intent == INTENT_HELP)
+		user.audible_message("<span class='emote'>[src] gives [M] a hug!</span>")
```

Comment:
```suggestion
		user.visible_message("<span class='emote'>[src] gives [M] a hug!</span>")
```

Correct me if I'm wrong but I believe this should be `visible_message` or else deaf people will be banned from receiving hugs

Reply:
If it's visible then blind people will be banned from receiving hugs!

Reply:
Some sort of else condition outputting a message about feeling something hugging the target, perhaps?